snippets/findify-autocomplete-element.liquid

```diff
@@ -0,0 +1,24 @@
+<!-- 
+    findify-autocomplete-element contains an empty HTML element which will be populated with corresponding SSR by renderApi with sections/findify-autocomplete.liquid
+-->
+
+{{ 'findify-product-card.css' | asset_url | stylesheet_tag }}
```

Comment:
Move this one into findify-head.snippet

If not everytime we make a autocomplete rendering it will be loaded and gonna be heavier on connection redownloading it every time